### PR TITLE
add fundraising

### PR DESCRIPTION
**Pull request by jukent**
Draft PR to demonstrate embedding fundraising on the site.

We can have more conversations about the text or where it should go. Personally, I like the way the UCAR page has the Donate button prominently next to the text (instead of hidden at the bottom) and the donate form is a pop up modal. This just a proof of concept.

### CODE REVIEW

File: portal/posts/fundraiser.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+blogpost: true
+date: Jun 28, 2023
+author: Julia kent
+tags: fundraiser
+---
+
+# Donate to Support Project Pythia!
+
+## You can make an impact on our community!
+By donating to support Project Pythia you are investing in an important educational resource for the entire geoscience community. Project Pythia is an education working group helping geoscientists make sense of huge volumes of numerical scientific data using tools that facilitate open, reproducible science, and building an inclusive community of practice around these goals. Project Pythia is a home for Python-centered learning resources that are open-source, community-owned, geoscience-focused, and high-quality.
```
Comment (clyne) [inline portal/posts/fundraiser.md:11]:
```suggestion
By donating to support Project Pythia you are investing in an important educational resource for the entire geoscience community, from students to late career. Project Pythia is an education working group helping geoscientists make sense of huge volumes of numerical scientific data using tools that facilitate open, reproducible science, and building an inclusive community of practice around these goals. Project Pythia is a home for Python-centered learning resources that are open-source, community-owned, geoscience-focused, and high-quality.
```

**Comment by jukent** [comment]
@clyne and @brian-rose are there any stylistic or other changes you'd like to see to the donation button or to the donation page?

**Comment by brian-rose** [comment]
I guess I consider the styling of the button on the portal landing page to be temporary since we are in the process of migrating to MyST and there is likely to be significant changes to the portal page aesthetics. As a "for now" solution, I think this is fine.

My main suggestion is about the donation page itself. I think it would be helpful to give some idea of what the funds might be used for. I'm not sure if we have a detailed plan for this, but additional participant support for our annual hackathons definitely comes to mind.

**Comment by jukent** [comment]
> My main suggestion is about the donation page itself. I think it would be helpful to give some idea of what the funds might be used for. I'm not sure if we have a detailed plan for this, but additional participant support for our annual hackathons definitely comes to mind.

That makes sense. Should brainstorm in the ad hoc Pythia meeting tomorrow?

**Comment by clyne** [comment]
I agree with @brian-rose on a statement about what the funds will be used for. In addition to participant support at hackathons, I would add outreach and community engagement activities. Thanks for putting this together so quickly @jukent !

**Review by clyne — APPROVED** [review]
Looks great, @jukent. Thank you

**Comment by jukent** [comment]
@brian-rose do you like the new addition on how funding is spent?

**Review by brian-rose — APPROVED** [review]
Looks great!

**Pull request merged**